api/app/Policies/PoolPolicy.php

```diff
@@ -75,40 +76,48 @@ public function viewAnyPublished(?User $user)
      */
     public function create(User $user, $request)
     {
-        if (array_key_exists('team_id', $request)) {
-            $team_id = $request['team_id'];
 
-            // Get the team to check against
-            $team = Team::find($team_id);
+        $teamId = isset($request['team_id']) ? $request['team_id'] : null;
+        $communityId = isset($request['community_id']) ? $request['community_id'] : null;
 
-            // Confirm the user can create pools for the team
-            if (! is_null($team)) {
-                if ($user->isAbleTo('create-team-draftPool', $team)) {
-                    return true;
-                }
-            } else {
-                return Response::deny('Cannot find a team matching team_id.');
-            }
-        } else {
-            Response::deny('Pool must be associated with a team when it is created.');
+        if (is_null($teamId) || is_null($communityId)) {
+            return false;
+        }
+
+        if ($user->isAbleTo('create-any-pool')) {
+            return true; // return early, permission does not exist at the moment
         }
 
-        return Response::deny('Cannot create a pool for that team.');
+        $team = Team::findOrFail($teamId);
+        $community = Community::findOrFail($communityId)->load('team');
+
+        if ($user->isAbleTo('create-team-draftPool', $team)) {
+            // user is a legacy pool operator
+            return true;
+        }
+
+        if (! is_null($community->team) && $user->isAbleTo('create-team-draftPool', $community->team)) {
+            // user is a community recruiter or community admin
+            return true;
+        }
+
+        return false; // fallback to fail
     }
 
     /**
-     * Determine whether the user can create pools.
+     * Determine whether the user can duplicate pools.
      *
      * @return \Illuminate\Auth\Access\Response|bool
      */
     public function duplicate(User $user, $request)
     {
-        $existing = Pool::findOrFail($request['id']);
+        $existing = Pool::findOrFail($request['id'])->load(['team', 'community.team']);
```

Comment:
Same here, use `with` instead of `load`.

Reply:
Sure can do, https://github.com/GCTC-NTGC/gc-digital-talent/pull/11070/commits/66cbcfb3ee0747a6232cb13337067ad430eb9379